src/Element_TEST.cc

```diff
@@ -37,6 +37,8 @@ TEST(Element, Child)
   sdf::Element child;
   sdf::ElementPtr parent = std::make_shared<sdf::Element>();
   parent->SetFilePath("/parent/path/model.sdf");
+  parent->SetLineNumber(12);
+  parent->SetXmlPath("/sdf/world[@name=\"default\"]");
```

Comment:
I've removed them since they are unrelated to that particular test case, https://github.com/osrf/sdformat/pull/548/commits/7aa851917475ba2ebb5628cee38034143501b48a, thanks for spotting it!